### PR TITLE
Audio `seqplayer.c`: AudioSeq_ to AudioScript_

### PR DESCRIPTION
**Pull request by engineer124**
A complementary PR to https://github.com/zeldaret/mm/pull/1201. This changes the prefix of audio's `seqplayer.c` from `AudioSeq_` to `AudioScript_` so that `AudioSeq_` can be used by `sequence.c`

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]
I would say those renames need to be added to namefixer, but since you are basically the only one which works with audio here it probably doesn't matter as long as it isn't a problem for you

**Comment by engineer124** [comment]
> I would say those renames need to be added to namefixer, but since you are basically the only one which works with audio here it probably doesn't matter as long as it isn't a problem for you

Every call to these functions are also decompiled and in master as well. `audio/lib` is fully decompiled and it's just the external audio left, which never access these internal functions :)

File: src/audio/lib/playback.c [hunk 1]
```diff
@@ -867,7 +867,7 @@ void AudioPlayback_NoteReleaseAndTakeOwnership(Note* note, SequenceLayer* layer)
 }
 
 Note* AudioPlayback_AllocNoteFromDisabled(NotePool* pool, SequenceLayer* layer) {
-    Note* note = AudioSeq_AudioListPopBack(&pool->disabled);
+    Note* note = AudioScript_AudioListPopBack(&pool->disabled);
     if (note != NULL) {
```
Comment (hensldm) [inline src/audio/lib/playback.c:871]:
```suggestion
    Note* note = AudioScript_AudioListPopBack(&pool->disabled);
    
    if (note != NULL) {
```

**Review by hensldm — APPROVED** [review]
1 small nit I saw

**Review by isghj5 — APPROVED** [review]

**Pull request merged**